### PR TITLE
Using the new facts hash instead of the global var in repo/debian.pp

### PR DESCRIPTION
**Pull request by c33s**
adde PR for https://github.com/voxpupuli/puppet-php/pull/420#pullrequestreview-100970274

### CODE REVIEW

File: manifests/repo/debian.pp [hunk 1]
```diff
@@ -86,7 +86,7 @@
 
     ::apt::source { 'source_php_71':
       location => 'https://packages.sury.org/php/',
-      release  => $::lsbdistcodename,
+      release  => $::facts['os']['distro']['codename'],
```
Comment (bastelfreak) [inline manifests/repo/debian.pp]: please remove the ::. They are not needed anymore.

**Comment by bastelfreak** [comment]
Hi @c33s, thanks for the PR. Can you take a look at the inline comment I made? Afterwards we can merge this.

**Comment by bastelfreak** [comment]
thanks!

**Comment by c33s** [comment]
@bastelfreak thank you for your hard work. i have seen you have been quite busy in this repo. keep up the good work.

**Pull request merged**